.github/workflows/ci.yml

```diff
@@ -28,4 +28,6 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
           bundler: latest
+      - run: git config --global user.name 'github-actions[bot]'
+      - run: git config --global user.email 'github-actions[bot]@users.noreply.github.com'
```

Comment:
🗒️ The integration tests now need to run actual `git commit` commands. In order to create commits, git must be configured with a user name and email.